en/docs/catalogs/kubernetes-crds.md

```diff
@@ -1 +1,4 @@
 # Kubernetes CRD Catalog
+
+WSO2 API management for Kubernetees now offers comprehensive support for the [Kubernetes Gateway API specification](https://gateway-api.sigs.k8s.io/). This support signifies a significant advancement in our API management for Kubernetees solution, enhancing our compatibility and integration capabilities within Kubernetes environments.
```

Comment:
shall we add the list of crds that we support here and link the following doc pages